lib/LLVMIRCodeGen/libobj/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Archiving additional objects within the bundle object
+
+All the object files (.o) stored in this directory will be automatically
+serialized as C buffer arrays and stored within Glow library at build time.
+When building bundles with Glow tools the user can opt to archive additional
+objects within the bundle object file by specifying the following command line
```

Comment:
Why do you consider only object files and not e.g. some other resource files that maybe needed by the bundle?

Reply:
The files must be object files because the intention is to archive them together with the original bundle created by Glow into a static library (object archive). Are there any file types I should consider?